chassis/spring.schema/src/test/java/com/griddynamics/jagger/xml/JaggerProvidersTest.java

```diff
@@ -21,16 +25,27 @@
 public class JaggerProvidersTest {
 
     private ApplicationContext ctx;
+    private H2DatabaseServer dbServer;
 
     @BeforeClass
     public void testInit() throws Exception{
         URL directory = new URL("file:" + "../configuration/");
         Properties environmentProperties = new Properties();
         JaggerLauncher.loadBootProperties(directory, "profiles/local/environment.properties", environmentProperties);
         environmentProperties.put("chassis.master.configuration.include",environmentProperties.get("chassis.master.configuration.include")+", ../spring.schema/src/test/resources/example-providers.conf.xml1");
+
+        ApplicationContext rdbContext = JaggerLauncher.loadContext(directory, RDB_CONFIGURATION, environmentProperties);
+        dbServer = (H2DatabaseServer) rdbContext.getBean("databaseServer");
+        dbServer.run();
+
         ctx = JaggerLauncher.loadContext(directory,"chassis.master.configuration",environmentProperties);
     }
 
+    @AfterClass
+    public void testShutdown() {
+        dbServer.terminate();
```

Comment:
I thought about it, but if you noticed, some local vars are needed for starting db, so all tests need to be refactored a lot. It'll take much time, I'm not sure if we need to spend this time on such bugs.